src/host/hub.c

```diff
@@ -365,7 +407,16 @@ static void connection_get_status_complete (tuh_xfer_t* xfer)
     hub_port_clear_feature(daddr, port_num, HUB_FEATURE_PORT_CONNECTION_CHANGE, connection_clear_conn_change_complete, 0);
   }else
   {
-    // Other changes are: Enable, Suspend, Over Current, Reset, L1 state
+    // Clear other port status change interrupts. TODO Not currently handled - just cleared.
+    if (p_hub->port_status.change.port_enable)
```

Comment:
can you add bracket `{ }` to if-else, it is minor, but I try to avoid if-else on newline without brackets.

Reply:
right ofcourse, Ill fix this soon

Reply:
Fixed